.github/workflows/unit-tests.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
       - name: Configure PHP
         run: |
-          if [ -z "$LOG_COVERAGE" ]; then rm /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini ; fi
+          if [ -z "$LOG_COVERAGE" ]; then rm /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini || true ; fi
```

Comment:
could use to only return true for 8.0
```
.... || [ `php -v` -eq 8.0 ]
```

Reply:
addresses, but otherwise prefer to test for GH action job var...